README.md

```diff
@@ -69,3 +81,74 @@ Pirsch::track(
     ],
 );
 ```
+
+### Filter pages
+
+You can configure the `TrackPageview` middleware to exclude specific pages from being tracked.
+
+On a specific rouute, you can exclude pages by adding a `except` property to the middleware class:
+
+```php
+use Pirsch\Http\Middleware\TrackPageview;
+
+Route::middleware(TrackPageview::class.':url/to/exclude/*')->group(function () {
+    Route::get('/', function () {
+        return view('welcome');
+    });
+});
+```
+
+Multiple urls can be excluded by separating them with a comma:
+
+```php
+use Pirsch\Http\Middleware\TrackPageview;
+
+Route::middleware(TrackPageview::class.':url/to/exclude/*,url/to/exclude2/*')->group(function () {
+    Route::get('/', function () {
+        return view('welcome');
+    });
+});
+```
+
+To exclude pages globally, you can create a new middleware that extends the `TrackPageview` middleware and add an `except` property:
+
+```php
+namespace App\Http\Middleware;
+
+use Pirsch\Http\Middleware\TrackPageview as Middleware;
+
+class TrackPageview extends Middleware
+{
+    /**
+     * The URIs that should be excluded from tracking.
+     *
+     * @var array<int,string>
+     */
+    protected array $except = [
+        'url/to/exclude/*',
+    ];
+
+    /**
+     * The Headers that should be excluded from tracking.
+     *
+     * @var array<int,string>
+     */
+    protected array $exceptHeaders = [
+        'X-ExcludedHeader',
+    ];
+}
+```
+
+- `except` is an array with all URIs paths taht you want to exclude from tracking.
+- `exceptHeaders` is an array with all Headers that you want to exclude from tracking.
+
+Then replace the `TrackPageview` middleware with this one on your `bootstrap/app.php` middleware configuration:
+
+```diff
+  ->withMiddleware(function (Middleware $middleware) {
+      $middleware->web(append: [
+-         \Pirsch\Http\Middleware\TrackPageview::class,
++         \App\Http\Middleware\TrackPageview::class,
```

Comment:
Unless the middleware is published your name space should probably remain `\Pirsch\Http\Middleware\TrackPageview`.

And one other small note is that this registration I think will only work with Laravel 11. 

Laravel 10 and below would be in `\App\Http\Kernel`

You may need to bump the composer.json minimum framework version requirement.